adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -96,8 +96,22 @@ - (void)initWithAppId:(nonnull NSString *)appId
   //Disable refresh functionality for all banners
   [[VungleSDK sharedSDK] disableBannerRefresh];
 
+  // Set init options for priority placement
+  NSMutableDictionary *initOptions = [NSMutableDictionary dictionary];
+  if (delegate) {
+    NSString *priorityPlacementID = delegate.desiredPlacement ? delegate.desiredPlacement : @"";
```

Comment:
@ashinagawa Hi Aki, I have checked the `desiredPlacement.length` in `GADMAdapterVungleInterstitial` and `GADMAdapterVungleRewardedAd` classes, I believe we don't need to check `delegate.desiredPlacement is nil ` here. Thanks a lot. ^_^

Reply:
Updated ;-)